app/models/consistency_checklist.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+class ConsistencyChecklist < ApplicationRecord
+  belongs_to :planning_application
+
+  with_options if: :complete? do
+    validate :description_matches_documents_determined
+    validate :proposal_details_match_documents_determined
+    validate :documents_consistent_determined
+    validate :description_change_requests_closed
+    validate :additional_document_requests_closed
+  end
+
+  enum status: { in_assessment: 0, complete: 1 }, _default: :in_assessment
+
```

Comment:
I've made the `description_matches_documents` , `documents_consistent` and `proposal_details_match_documents` fields enums rather than booleans because they need three states - not answered, answered 'Yes', and answered 'No'. 

Elsewhere we've used booleans for similar scenarios and relied on states of `nil`, `true`, and `false`, but it seems to expose us to ambiguity and bugs because ruby often treats `nil` and `false` as equivalent. Eg if I did that here `consistency_checklist.description_matches_documents?` would return `false` when the column contained either `nil` or `false`.

I'd be interested to know if this is an approach people like or not!